app/assets/javascripts/discourse/mixins/add-category-class.js.es6

```diff
@@ -11,7 +11,7 @@ export default {
 
     this._removeClass();
 
-    if (categoryFullSlug) {
+    if (categoryFullSlug && this.get("categoryBackgroundUrl")) {
```

Comment:
If we do this, people who don't use a background image for their category won't be able to make some customisation based on the category class.